packages/api/src/server.ts

```diff
@@ -17,6 +20,40 @@ export class WepublishServer {
   constructor(opts: WepublishServerOpts) {
     const app = express()
 
+    const {dbAdapter} = opts
+
+    methodsToProxy.forEach(mtp => {
+      if (mtp.key in dbAdapter) {
+        mtp.methods.forEach(method => {
+          const methodName = `${method}${capitalizeFirstLetter(mtp.key)}`
+          // @ts-ignore
```

Comment:
Did you leave those `@ts-ignore` on purpose here?

Reply:
Yes. If i leave the out typescript complains and at the moment i have no idea how to type this.

Reply:
Think would have a fix there, will push later.